apps/hip/templates/includes/header.html

```diff
@@ -65,7 +65,7 @@
                 <i class="signin-icon-hip"></i>
               </span>
               <span class="button is-size-7-touch header-btn-hip ml-0 pr-6">
```

Comment:
I think we should `center` the text for this button or make the button's width slightly smaller.

Reply:
Sure, that sounds good. It looks like removing the extra padding on the right accomplishes this